### PR TITLE
Update NuxtJS Link to Current Version

### PR DESCRIPTION
**Pull request by they-cloned-me**
### Description
Update outdated link to **NuxtJS**, from `https://nuxtjs.org` (**v2**) to `https://nuxt.com/` (**v3**).

### Motivation
For users reading the docs and/or following along with tutorials, it is important they get pointed to the right learning resources.

### CODE REVIEW

File: files/en-us/learn/tools_and_testing/client-side_javascript_frameworks/vue_resources/index.md [hunk 1]
```diff
@@ -50,7 +50,7 @@ Here's where you should go to learn more about Vue:
 - [Vue GitHub Repo](https://github.com/vuejs/vue) — The Vue code itself. This is where you can report issues and/or contribute directly to the Vue codebase. Studying the Vue source code can help you better understand how the framework works, and write better code.
 - [Vue Forum](https://forum.vuejs.org/) — The official forum for getting help with Vue.
 - [Vue CLI Docs](https://cli.vuejs.org/) — Documentation for the Vue CLI. This contains information on customizing and extending the output you are generating via the CLI.
-- [NuxtJS](https://nuxtjs.org/) — NuxtJS is a Server-Side Vue Framework, with some architectural opinions that can be useful to creating maintainable applications, even if you don't use any of the Server Side Rendering features it provides. This site provides detailed documentation on using NuxtJS.
+- [NuxtJS](https://nuxt.com/) — NuxtJS is a Server-Side Vue Framework, with some architectural opinions that can be useful to creating maintainable applications, even if you don't use any of the Server Side Rendering features it provides. This site provides detailed documentation on using NuxtJS.
```
Comment (danielroe) [inline files/en-us/learn/tools_and_testing/client-side_javascript_frameworks/vue_resources/index.md:53]:
```suggestion
- [Nuxt](https://nuxt.com/) — NuxtJS is a Server-Side Vue Framework, with some architectural opinions that can be useful to creating maintainable applications, even if you don't use any of the Server Side Rendering features it provides. This site provides detailed documentation on using NuxtJS.
```

File: files/en-us/learn/tools_and_testing/client-side_javascript_frameworks/introduction/index.md [hunk 2]
```diff
@@ -294,7 +294,7 @@ A few popular CMS systems include [Wordpress](https://wordpress.com/), [Joomla](
...
 
-All of the frameworks covered in this module support server-side rendering as well as client-side rendering. Check out [Next.js](https://nextjs.org/) for React, [Nuxt.js](https://nuxtjs.org/) for Vue (yes, it is confusing, and no, these projects are not related!), [FastBoot](https://github.com/ember-fastboot/ember-cli-fastboot) for Ember, and [Angular Universal](https://angular.io/guide/universal) for Angular.
+All of the frameworks covered in this module support server-side rendering as well as client-side rendering. Check out [Next.js](https://nextjs.org/) for React, [Nuxt.js](https://nuxt.com/) for Vue (yes, it is confusing, and no, these projects are not related!), [FastBoot](https://github.com/ember-fastboot/ember-cli-fastboot) for Ember, and [Angular Universal](https://angular.io/guide/universal) for Angular.
```
Comment (danielroe) [inline files/en-us/learn/tools_and_testing/client-side_javascript_frameworks/introduction/index.md:297]:
```suggestion
All of the frameworks covered in this module support server-side rendering as well as client-side rendering. Check out [Next.js](https://nextjs.org/) for React, [Nuxt](https://nuxt.com/) for Vue (yes, it is confusing, and no, these projects are not related!), [FastBoot](https://github.com/ember-fastboot/ember-cli-fastboot) for Ember, and [Angular Universal](https://angular.io/guide/universal) for Angular.
```

File: files/en-us/learn/tools_and_testing/client-side_javascript_frameworks/vue_resources/index.md (hunk 1 above)
Comment (danielroe) [reply files/en-us/learn/tools_and_testing/client-side_javascript_frameworks/vue_resources/index.md:53]: Thank _you_ for this PR.

**Review by hamishwillee — APPROVED** [review]
Thanks very much @they-cloned-me and @danielroe . 

FYI @they-cloned-me - you can merge suggestions to your own PRs if you think they make sense. That way the reviewer is presented with a consolidated view.

**Comment by they-cloned-me** [comment]
> Thanks very much @they-cloned-me and @danielroe .
> 
> FYI @they-cloned-me - you can merge suggestions to your own PRs if you think they make sense. That way the reviewer is presented with a consolidated view.

Awesome, thank you! 💚

**Pull request merged**